### PR TITLE
GG-19492 Add debug output for org.apache.ignite.internal.processors.cache.persistence.GridCacheOffheapManager#restorePartitionStates method.

### PR DESCRIPTION
**Pull request by 6uest**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java [hunk 1]
```diff
@@ -569,13 +573,18 @@ else if (recoverState != null) { // Pre-create partition if having valid state.
...
             }
             else {
                 if (log.isDebugEnabled())
                     log.debug("Skipping partition on recovery (no page store OR wal state) " +
                         "[grp=" + grp.cacheOrGroupName() + ", p=" + p + "]");
             }
+            if (log.isDebugEnabled())
+                log.debug("Finished restoring partition state " +
+                    "[grp=" + grp.cacheOrGroupName() + ", p=" + p +
+                    ", time=" + (U.currentTimeMillis() - startTime) + "ms]");
```
Comment (zstan) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java]: plz add whitespace before "ms"

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java [hunk 2]
```diff
@@ -569,13 +573,18 @@ else if (recoverState != null) { // Pre-create partition if having valid state.
...
-                        ", updCntr=" + part.initialUpdateCounter() + "]");
+                        ", updCntr=" + part.initialUpdateCounter() +
+                        ", size=" + part.fullSize() + "]");
             }
             else {
                 if (log.isDebugEnabled())
                     log.debug("Skipping partition on recovery (no page store OR wal state) " +
                         "[grp=" + grp.cacheOrGroupName() + ", p=" + p + "]");
             }
+            if (log.isDebugEnabled())
```
Comment (zstan) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java]: new line need to be here? isn`t it ?

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java (hunk 1 above)
Comment (6uest) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java]: @zstan done.

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java (hunk 2 above)
Comment (6uest) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java]: @zstan done

**Comment by dspavlov** [comment]
The ticket is resolved, closing PR.  
Feel free to reopen this PR if it is anyway actual.

**Pull request closed without merging**